src/pybind/mgr/prometheus/module.py

```diff
@@ -1267,13 +1266,20 @@ def _get_pool_info(pool: Dict[str, Any]) -> Tuple[str, str]:
             if 'options' in pool:
                 compression_mode = pool['options'].get('compression_mode', 'none')
 
+            application_metadata_str = ''
+            if not(pool['pool_name'].startswith('.')):
```

Comment:
```suggestion
            if not pool['pool_name'].startswith('.'):
```
to address
```
prometheus/module.py:1270:19: E275 missing whitespace after keyword
1     E275 missing whitespace after keyword
```

Reply:
I figured out the logic was not right, I changed and handled the filtration in UI layer